tiny_hippo_run.py

```diff
@@ -52,9 +52,12 @@ def main():
 
     # 4) Capture IoT packets only with crafted sniff
     print("Capturing IoT packets only")
-    # TODO: Make sure iface is set to the correct interface. May be different in some routers
-    sniff(iface="wlan0", lfilter=lambda packet: (packet.src in mac_addrs) or (packet.dst in mac_addrs),
-          prn=packet_parse, count=num_packets)
+    network_interface = run_config.network_interface
+    try:
+        sniff(iface=network_interface, lfilter=lambda packet: (packet.src in mac_addrs) or (packet.dst in mac_addrs),
+              prn=packet_parse, count=num_packets)
+    except Exception as e:
+        run_config.log_event.info('Could not sniff packet. Exception: ' + str(e))
```

Comment:
Should we do a re-throw or some output to console that the issue has been logged here so we don't just crash the program without any real indication of an issue?

Reply:
That would probably be a good idea. That begs the question, what is the best way for the sniff function to continue sniffing if it fails for some reason? Put it in a while loop? Any suggestions would be appreciated.